src/mixins/Projector.ts

```diff
@@ -122,6 +123,18 @@ const eventHandlers = [
 	'onsubmit'
 ];
 
+/**
+ * Internal function that maps existing DOM Elements to virtual DOM nodes.
+ * @param vnode The virtual DOM node
+ * @param domNode The Element, if any, to set on the virtual DOM node
+ */
+function setDomNodes(vnode: VNode, domNode: Element | null = null) {
+	vnode.domNode = domNode;
+	if (vnode.children && domNode) {
```

Comment:
The code is written in a way to deal with subsuming or not subsuming an existing DOM structure.  When merging, it is attempted to stitch together the existing DOM nodes, but if for some reason there is part of the DOM missing, it would result in a run time error.  Any missing DOM will be created by maquette.